reflex/utils/exec.py

```diff
@@ -155,3 +157,50 @@ def run_backend_prod(
         str(num_workers),
     ]
     processes.new_process(command, run=True, show_logs=True)
+
+
+def output_system_info():
+    """Show system informations if the loglevel is in DEBUG."""
+    if console.LOG_LEVEL > constants.LogLevel.DEBUG:
+        return
+
+    console.rule(f"System Info")
+
+    dependencies = [
+        f"[Reflex {constants.VERSION} with Python {platform.python_version()} (PATH: {sys.executable})]",
+        f"[Node {prerequisites.get_node_version()} (Expected: {constants.NODE_VERSION}) (PATH:{constants.NODE_PATH})]",
+        f"[NVM {constants.NVM_VERSION} (Expected: {constants.NVM_VERSION}) (PATH: {constants.NVM_PATH})]",
+    ]
+
+    system = platform.system()
+
+    if system == "Linux":
+        import distro
+
+        os_version = distro.name(pretty=True)
+    else:
+        os_version = platform.version()
+
+    if system != "Windows":
+        dependencies.append(
+            f"[Bun {prerequisites.get_bun_version()} (Expected: {constants.BUN_VERSION}) (PATH: {constants.BUN_PATH})]",
+        )
+
+    dependencies.append(f"[OS {platform.system()} {os_version}]")
+
+    for dep in dependencies:
+        console.debug(f"{dep}")
+
+    if system == "Windows":
+        console.debug(
+            f"Using package manager at : {prerequisites.get_windows_package_manager()}"
+        )
+    else:
+        console.debug(
+            f"Using package installer at : {prerequisites.get_install_package_manager()}"
```

Comment:
This method handles the Windows case also

Reply:
I see.
But we still need different output anyway because Windows use npm for both.

Reply:
refactored this now that `get_windows_package_manager` is no longer used